app/controllers/users_controller.rb

```diff
@@ -107,17 +92,17 @@ def self.unimportant_changes?(resource, params)
       (params[:current_password].blank? &&
           params[:password].blank? &&
           params[:password_confirmation].blank? &&
-          params[:email] == resource[:email])
+          (params[:email].blank? || params[:email] == resource[:email]))
     end
 
     # Overridden methods of `Devise::RegistrationsController` to permit additional model params
     def sign_up_params
       generate_random_password if get_omniauth_data
-      params.require(:user).permit(:first_name, :last_name, :email, :password, :image, :remove_image, :password_confirmation, event_ids: [])
+      params.require(:user).permit(:first_name, :last_name, :email, :password, :image, :remove_image, :password_confirmation, :avatar, :remove_avatar, :birthday, :telephone_number, :telegram_username, :favourite_sports, event_ids: [])
     end
 
     def account_update_params
-      params.require(:user).permit(:first_name, :last_name, :email, :password, :password_confirmation, :current_password, event_ids: [])
+      params.require(:user).permit(:first_name, :last_name, :email, :password, :password_confirmation, :current_password, :avatar, :remove_avatar, :birthday, :telephone_number, :telegram_username, :favourite_sports, event_ids: [])
```

Comment:
Would it make sense to split up normal profile and password updates? otherwise one would always have to confirm a profile update with a password

Reply:
there was a userstory that made a password confirmation only necessary for important changes. This feature already works, no worries :)